src/Microsoft.Azure.WebJobs.Host/Bindings/BindingExpressionExtensionConfigProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Extension provider for <see cref="BindingExpressionAttribute"/>
+    /// </summary>
+    internal class BindingExpressionExtensionConfigProvider : IExtensionConfigProvider
+    {
+        public void Initialize(ExtensionConfigContext context)
+        {
+            var rule = context.AddBindingRule<BindingExpressionAttribute>();
```

Comment:
I don't think these input bindings will be immediately accessible to script functions without changes to the runtime. Have you tried this end to end yet?